### PR TITLE
updated release notes for API-3044 feature

### PR DESCRIPTION
**Pull request by gpbowman-git**
### Description
Updating Benefits Intake release notes tied to API-3044. This feature added PDF upload information to a DB column and included a JSON response in the status check related to the documents contained in the uploaded payload

https://vajira.max.gov/browse/API-3044

### Process

- [ ] Tests added or updated for change
- [ ] Docs updated
  - [ ] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [ ] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [ ] Accessibility concerns have been considered and addressed
  
  - [ ] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### CODE REVIEW

**Review by rtravitz — DISMISSED** [review]
Thanks for adding a release note. This works from a functional standpoint.

File: src/content/apiDocs/benefits/benefitsIntakeReleaseNotes.mdx [hunk 1]
```diff
@@ -1,3 +1,10 @@
+### November 30, 2020
+(v1) Added uploaded PDF metadata to the DB and in the HTTP response:
```
Comment (ryanlink) [inline src/content/apiDocs/benefits/benefitsIntakeReleaseNotes.mdx:2]: Sorry for the late review - may I suggest adding "(document count and page count)" after "metadata" so it's clear what metadata was added. Thx!

**Review by ryanlink — CHANGES_REQUESTED** [review]

File: src/content/apiDocs/benefits/benefitsIntakeReleaseNotes.mdx (hunk 1 above)
Comment (gpbowman-git) [reply src/content/apiDocs/benefits/benefitsIntakeReleaseNotes.mdx]: I have added additional information in the description. Let me know if this works for you.

**Review by ryanlink — APPROVED** [review]
LGTM!

**Review by mjlumetta — APPROVED** [review]
lgtm, feel free to merge!

**Pull request merged**